x-pack/test/alerting_api_integration/common/lib/es_test_index_tool.ts

```diff
@@ -4,20 +4,18 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const ES_TEST_INDEX_NAME = '.kibaka-alerting-test-data';
+export const ES_TEST_INDEX_NAME = '.kibana-alerting-test-data';
```

Comment:
been meaning to change this for a while :-)

Reply:
❤️ go kibaka!